defi/src/utils/sluggify.test.ts

```
@@ -0,0 +1,10 @@
+import { sluggifyString } from './sluggify';
+
+const tests: Record<string, string> = {
+  've(3,3)': 've33',
+  'Prediction Market': 'prediction-market',
+};
+
+test("should get correct slug string", () => {
+  Object.keys(tests).forEach(t => expect(sluggifyString(t)).toEqual(tests[t]))
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)